docs/pages/en-us/Features.md

```diff
@@ -13,29 +13,24 @@ banner:
 
 # Features
 
-The Audit service maintains a documented audit trail of user and system activity in a centralized database. The benefits of using the Audit service include traceability and accountability, but the audit records can also be used for analytical, archiving and reporting purposes. 
+The Audit service maintains a documented audit trail of user and system activity in a centralized database. The benefits of using the Audit service include traceability and accountability, but the audit records can also be used for analytical, archiving and reporting purposes.
 
 ## Support for Multiple Applications
-User and system audit events are defined per application. The Audit service offers an easy and user-friendly way to register these events using the Audit Management web service API.
+User and system audit events are defined per application.
 
 ## Multi-Tenant Aware
-The service supports multiple tenants in a secure and scalable way. These tenants are registered using the Audit Management web service API. Multi-tenancy support also facilitates audit reporting on a per tenant basis. 
+The service supports multiple tenants in a secure and scalable way.  Multi-tenancy support also facilitates audit reporting on a per tenant basis. 
 
 ## Auto-Generated Client Library
-The service includes an auto-generated client library for type safety. This client library also makes it easier to send user and system audit events to the messaging system.
-
-## High-Throughput Messaging
-The service uses Apache Kafka internally. Apache Kafka's high-throughput distributed messaging system is designed to be fast.
+The service includes an auto-generated client-side library for type safety.  This client library also makes it easier to send user and system audit events to Elasticsearch.
 
 ## Scalable
-The service is designed for scalability as the Kafka cluster can elastically and transparently expand without downtime. Data streams are partitioned over a cluster of machines to provide streams much larger than those available through a single machine.
+The service is extensively for scalabe as the Elasticsearch is designed with scalability in mind and can scale horizontally without downtime.
```

Comment:
Grammar and spelling correction here. I would change it to something like: "The service is extensively scalable. Elasticsearch is designed with scalability in mind and can scale horizontally without downtime."